v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/source/reader/io/jdbc/rowmapper/JdbcSourceRowMapperTest.java

```diff
@@ -323,7 +333,7 @@ private static ImmutableList<Col> getTestCols() {
                     "medium_blob_col",
                     "BLOB",
                     new SourceColumnType("MEDIUMBLOB", new Long[] {10L}, null))
-                .withValue(new byte[] {0x65, 0x66}, new char[] {'6', '5', '6', '6'})
+                .withValue(new byte[] {0x65, 0x66}, "6566")
                 .build())
```

Comment:
Similarly UTs for binary, varbinay? And maybe YEAR too to catch this in UTs early on?

Reply:
We have a UT for bianry and varbinary, I have updated the same. The UT could not catch the issue earlier since we use derby and not actual db for testing.
For `year` we dont have a corresponding type in derby but we have an existing UT that maps it to dummy derby type (this wont be as effective as an actual DB run).